### PR TITLE
Update useChain.ts

### PR DESCRIPTION
**Pull request by Y0moo**
---
name: "Pull request"
about: A new pull request
---

### Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/MoralisWeb3/react-moralis/blob/main/SECURITY.md).
- [x] My code is conform the [code style](https://github.com/MoralisWeb3/react-moralis/blob/main/CODE_STYLE.md)
- [x] I have made corresponding changes to the documentation
- [x] I have updated Typescript definitions when needed

### Issue Description

New chains can't be added to the Metamask. `blockExplorerUrl` is not exported from chains

Fixes error message:
```
{code: -32602, message: "Expected null or array with at least one valid string HTTPS URL 'blockExplorerUrl'. Received: ", stack: '{\n  "code": -32602, message: "Expected null or array with at least one valid string HTTPS URL 'blockExplorerUrl'. Received: ")'}
```

### Solution Description

Import `infoURL` as a `blockExplorerUrl`

### CODE REVIEW

**Comment by ErnoW** [comment]
@Y0moo For what chain is this error occurring? 
infoURL is different than blockExplorerUrl. 
The blockExplorerUrls are defined in https://github.com/MoralisWeb3/react-moralis/blob/main/src/data/blockExplorers.ts and are appended to the chainData

**Comment by Y0moo** [comment]
@ErnoW 

I added fantom and changed explorer for Mumbai. Also deleted `infoURL` import

File: src/data/blockExplorers.ts [hunk 1]
```diff
@@ -9,5 +9,6 @@ export const blockExplorers: Record<string, string | null> = {
   "0x38": "https://bscscan.com/",
   "0x61": "https://testnet.bscscan.com",
   "0x89": "https://explorer-mainnet.maticvigil.com",
-  "0x13881": "https://rpc-mumbai.matic.today",
+  "0x13881": "https://mumbai.polygonscan.com",
+  "0xfa": "https://ftmscan.com/"
```
Comment (ErnoW) [inline src/data/blockExplorers.ts:13]:
```suggestion
  "0xfa": "https://ftmscan.com/",
```

**Comment by ErnoW** [comment]
@Y0moo pipeline was failing because of a formatting issue.
Make sure to have the Prettier extension installed for automatic formatting.
I will look into auto-formatting in the CI later on, but for now, it is something to be aware of to check if everything is formatted and no es-lint errors are present.
One missing comma and your PR will be rejected 😅 (I know it is a bit harsh)

For now, I fixed it already. Nice work 👍

**Review by ErnoW — APPROVED** [review]

**Pull request merged**